app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/add/FetchTestRecordFragment.kt

```diff
@@ -80,9 +80,11 @@ class FetchTestRecordFragment : Fragment(R.layout.fragment_fetch_covid_test_resu
                     showLoader(state.onLoading)
 
                     if (state.errorData != null) {
-                        requireContext().showError(
-                            getString(state.errorData.title),
-                            getString(state.errorData.message)
+                        AppAlertDialog.showConfirmationAlertDialog(
+                            context = requireContext(),
+                            title = getString(state.errorData.title),
+                            msg = getString(state.errorData.message),
+                            positiveBtnMsg = getString(android.R.string.ok)
```

Comment:
do not use system string resources please declare our own resource in string.xml

Reply:
Done